### ISSUE TITLE
[dextract] Error 1

### ISSUE THREAD

**Issue by taotaoyuan** [xiaochuanle/MECAT#76]
$make -f dextract_makefile
gcc -O3 -Wall -Wextra -Wno-unused-result -fno-strict-aliasing -I/data1/spider/ytbiosoft/soft/hdf5-1.8.15-patch1/hdf5/include -L/data1/spider/ytbiosoft/soft/hdf5-1.8.15-patch1/hdf5/lib -o dextract dextract.c DB.c QV.c -lhdf5
/tmp/ccxK8viC.o: In function `main':
dextract.c:(.text+0x144): undefined reference to `parse_filter'
dextract.c:(.text+0x25e): undefined reference to `initBaxData'
dextract.c:(.text+0x31f): undefined reference to `getBaxData'
dextract.c:(.text+0x345): undefined reference to `nextSubread'
dextract.c:(.text+0x35b): undefined reference to `nextSubread'
dextract.c:(.text+0x37c): undefined reference to `evaluate_bax_filter'
dextract.c:(.text+0x81f): undefined reference to `sam_close'
dextract.c:(.text+0x8f2): undefined reference to `sam_open'
dextract.c:(.text+0x908): undefined reference to `sam_header_process'
dextract.c:(.text+0x937): undefined reference to `sam_record_extract'
dextract.c:(.text+0x94a): undefined reference to `SAM_EOF'
dextract.c:(.text+0x95d): undefined reference to `evaluate_bam_filter'
dextract.c:(.text+0xee7): undefined reference to `sam_open'
dextract.c:(.text+0x12ab): undefined reference to `parse_filter'
dextract.c:(.text+0x12ef): undefined reference to `printBaxError'
collect2: ld returned 1 exit status
make: *** [dextract] Error 1

**Comment by yejilee-biostat** [comment]
I'm getting the same error message.

Edit: This reply from a previous issue was really helpful!: https://github.com/xiaochuanle/MECAT/issues/65#issuecomment-451365296

**Comment by taotaoyuan** [comment]
did you solve it?

**Comment by yejilee-biostat** [comment]
> did you solve it? @taotaoyuan 

Yes - after changing the dextractor_makefile as the previous link, the problem is solved.

**Comment by taotaoyuan** [comment]
What does the previous link represent?Can you be more specific?

**Comment by yejilee-biostat** [comment]
> What does the previous link represent?Can you be more specific?

Sorry - I saw this comment too late. Hope you have already fixed your problem, but again - the comment made by "githubwangqiqi" in the link from my first comment.
